package.json

```diff
@@ -2,26 +2,21 @@
   "name": "@kodingdotninja/chakra-icons",
   "version": "0.0.0",
   "workspaces": [
-    "packages/*",
-    "website",
-    "!website/**",
-    "!packages/tsconfig/**",
-    "!packages/chakra-icons/**",
-    "!packages/msrconfig/**"
+    "packages/**",
```

Comment:
Anyway, I have problem with this `packages/**` 

cause the yarn workspaces will deep the project tree and conflicted with `chakra-icons` generated. 

really need to deep the project in `packages` dir ? 

![image](https://user-images.githubusercontent.com/16365952/150234006-66af180f-f9a0-43cf-a460-04db7c59c2c6.png)

I can install the packages by changes workspaces to `packages/*`

Reply:
Makes sense. Initially using deep globs for scoped packages. Current globs should work for both regular and scoped packages.

Reply:
It's work but have a little error (nope). 

I have make commit too for `yarn validate` change to `yarn lint` in workflows.